Update expo-secure-store.json

typedashutosh opened the pull request:
Replaced "witch" with "with"

# Why

There was a spelling mistake at https://docs.expo.dev/versions/latest/sdk/securestore/#securestoreisavailableasync

# How

Replaced "witch" with "with"

# Test Plan

<!--
Please describe how you tested this change and how a reviewer could reproduce your test, especially if this PR does not include automated tests! If possible, please also provide terminal output and/or screenshots demonstrating your test/reproduction.
-->

# Checklist

- [ x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

typedashutosh committed:
Update expo-secure-store.json

Replaced "witch" with "with"

Simek commented:
Hello @typedashutosh, thank you for spotting that typo! Unfortunately, those files are automatically generated from the source files, and manual edit will be lost, if not performed in the package code. 

We have guide which you can follow to perform the changes as needed, if you would like to continue work on this fix:
* https://github.com/expo/expo/tree/main/docs#update-api-reference-docs

typedashutosh commented:
- I deleted the fork (I dont know why) then reforked it (I think i'll have to create a new pr, I'll see)
- ran `npm run setup:docs` which didnt worked, so manually ran the `./scripts/download-dependencies.sh` file
![image](https://github.com/expo/expo/assets/68853189/a19ad018-866b-44db-ae89-c7dd09cbb680)
![image](https://github.com/expo/expo/assets/68853189/84335523-c575-40cd-8c28-25ac8988bd4b)
- Then I had to install Ruby to run Expotools
- And I am sure I'll have to change the Spelling here in file `~\Documents\VSCode\expo\packages\expo-secure-store\src\SecureStore.ts`
![image](https://github.com/expo/expo/assets/68853189/ca9cefda-c26c-486f-a53c-886af192db83)
-Then I fixed the type and ran `expotools` which failed with `bash: expotools: command not found`
![image](https://github.com/expo/expo/assets/68853189/b0d36f1d-3ba2-41c7-b98d-9aceba4ecf13)
What should i do next?

typedashutosh commented:
Do I need to something with these scripts inside `~/Documents/VSCode/expo/packages/expo-secure-store` 
![image](https://github.com/expo/expo/assets/68853189/ec89da60-5501-48c6-a0d5-7a74faa560b3)
